index.js

```diff
@@ -1,60 +1,58 @@
-const { Client, Collection } = require('discord.js-light');
-const { token, db } = require('./src/config/config.json');
-const splash = require('star-splash');
-const handler = require('./src/handler/handler');
-console.log('[SHYNK] Iniciando o bot...');
+const { Client, Collection } = require('discord.js-light')
+const { token, db } = require('./src/config/config.json')
+const splash = require('star-splash')
+const handler = require('./src/handler/handler')
+console.log('[SHYNK] Iniciando o bot...')
 
 const client = new Client({
-	cacheGuilds: true,
-	cacheRoles: true,
-	cachePresences: false,
-	cacheChannels: false,
-	cacheOverwrites: false,
-	cacheEmojis: false,
-	disabledEvents: [
-		'TYPING_START',
-		'CHANNEL_DELETE',
-		'CHANNEL_CREATE',
-		'GUILD_BAN_ADD',
-		'GUILD_BAN_REMOVE',
-		'CHANNEL_PINS_UPDATE'
-	]
-});
-
-client.commands = new Collection();
-client.events = new Collection();
-client.aliases = new Collection();
+  cacheGuilds: true,
+  cacheRoles: true,
+  cachePresences: false,
+  cacheChannels: false,
+  cacheOverwrites: false,
+  cacheEmojis: false,
+  disabledEvents: [
+    'TYPING_START',
+    'CHANNEL_DELETE',
+    'CHANNEL_CREATE',
+    'GUILD_BAN_ADD',
+    'GUILD_BAN_REMOVE',
+    'CHANNEL_PINS_UPDATE'
+  ]
+})
+
+client.commands = new Collection()
+client.events = new Collection()
+client.aliases = new Collection()
 console.log('[SHYNK | Collection] Comandos, eventos e aliases foram carregados.')
 
-
 console.log('[SHYNK | Login] Tentando fazer o login com o Discord...')
-client.login(token);
+client.login(token)
 
 // Eventos iniciais
 client.once('ready', () => {
-	console.log(splash(`[SHYNK] ${client.user.tag} Online!`, 'magenta'));
-	console.log(splash(`[DATABASE] Ping:\n${splash('Base', 'red')} - ${db.ping.basewrite}ms\n${splash('File', 'red')} - ${db.ping.filewrite}ms`, 'blue'));
-});
+  console.log(splash(`[SHYNK] ${client.user.tag} Online!`, 'magenta'))
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Shibe-Studio/Shynk/pullRequest?prid=7549193)
```suggestion
  console.log(splash(`[SHYNK] ${client.user.tag} Online!`, "magenta"))
```